importer/validation.py

```diff
@@ -78,24 +81,32 @@ def __validate_pair_naming_convention_for_read(read: RawRead) -> List[str]:
     return result
 
 
+def check_double_ended_column_is_T_or_F(spreadsheet: Spreadsheet) -> List[str]:
```

Comment:
I would find "Y" and "N" more intuitive. Would also be good if this could be made case-insensitive.